src/workflow/CompareCommitToRevisionWorkflow.php

```diff
@@ -123,6 +129,15 @@ public function run() {
     }
 
 
+    if ($this->getArgument('repo-call-sign') &&
+        $this->getArgument('repo-phid')) {
+      echo pht(
+        'Error: You cannot specify both --repo-call-sign and --repo-phid. '.
+        'Please choose one or the other.'
+      );
+      return 1;
+    }
+
```

Comment:
Nit:
```suggestion
$repo_call_sign = $this->getArgument('repo-call-sign');
$repo_phid = $this->getArgument('repo-phid');

if ($repo_call_sign && $repo_phid) {
    echo pht('Error: You cannot specify both --repo-call-sign and --repo-phid. Please choose one or the other.');
    return 1;
}
```

and then 

```
// if a call-sign is specified, look up the repository PHID
if ($repo_call_sign) {
    $this->repositoryId = $this->getRepositoryPHIDFromCallSignConduit($repo_call_sign);
    if (!$this->repositoryId) {
        echo pht('No repository found for call sign %s. Exiting', $repo_call_sign);
 
```       return 1;
    }
} else if ($repo_phid) {
    $this->repositoryId = $repo_phid;
}

Reply:
good feedback!  Fixed.